src/components/QuickOpenModal.js

```diff
@@ -420,7 +420,7 @@ function mapStateToProps(state) {
 
   return {
     enabled: getQuickOpenEnabled(state),
-    sources: formatSources(getRelativeSources(state), getTabs(state)),
+    sources: formatSources(getRelativeSourcesList(state), getTabs(state)),
```

Comment:
this is a minor change, but i'd rather to the data munging in a selector which is easier to type and memoize.